build/lib/test/test.js

```diff
@@ -407,10 +412,13 @@ class DeviceTestDetails {
 	 */
 	handleLine(token, stripped) {
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/test/te...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/test/test.js;
  Line: 413;
-->

- :warning: build/lib/test/test.js line 413 – 'stripped' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)